packages/@react-types/table/src/index.d.ts

```diff
@@ -112,7 +123,14 @@ export interface SpectrumColumnProps<T> extends ColumnProps<T> {
    * will be displayed when the column header is focused instead. Note that this prop is specifically for columns
    * that contain ActionButtons in place of text content.
    */
-  hideHeader?: boolean
+  hideHeader?: boolean,
+  /**
+   * Custom actions to be displayed in a drop-down menu on the column header. These may appear in addition
+   * to sorting and resizing. A Section can be used to group actions together.
+   */
+  actions?: ReactElement<ReactFragment & {children: ItemElement<T> | SectionElement<T> | Array<ItemElement<T> | SectionElement<T>> | ItemRenderer<T>}> | ReactElement<SectionProps<T>>,
```

Comment:
I don't think TS actually type checks JSX elements?

Reply:
Ah, I was using https://github.com/adobe/react-spectrum/blob/main/packages/%40react-types/shared/src/collections.d.ts#L47 as a guide
You are right, though, TS doesn't actually check this. In that case I could change it to type ReactNode, but that feels a little disingenuous in the docs page.